app/utils/constants.js

```diff
@@ -5,3 +5,10 @@ export const VOTING_STATUS_REJECTED = 'VOTING_STATUS_REJECTED'
 export const VOTING_STATUS_ACCEPTED = 'VOTING_STATUS_ACCEPTED'
 export const VOTING_STATUS_PENDING_ENACTMENT = 'VOTING_STATUS_PENDING_ENACTMENT'
 export const VOTING_STATUS_ENACTED = 'VOTING_STATUS_ENACTED'
+export const VOTING_STATUS = {
+  VOTING_STATUS_ONGOING: '',
```

Comment:
Check the comment about using `Object.keys(VOTING_STATUS)` here for the rest of statuses